code/obj/item/pens_writing_etc.dm

```diff
@@ -378,6 +378,37 @@
 	New()
 		. = ..()
 		src.create_inventory_counter()
+	
+	attack(mob/M as mob, mob/user as mob, def_zone)
+		if (iscarbon(M) || ismobcritter(M))
+			if (M == user)
+				return // so you don't accidentally shove a crayon while drawing
+				       // if you want to add this, PLEASE make it obvious
+			else if(check_target_immunity (M))
+				user.show_message( "<span class='alert'>You jam [src] up [M]'s nose, but it wont!</span>")
```

Comment:
> .... but it wont!
i assume there's meant to be a wont fit?/go in?

> You jam [src] up....
probably could do with a attempt to. "You attempt to jam [src] up..."

Reply:
Ah, crap. Yeah, that's a typo. This only appears when trying to shove a crayon up the noses of admins, but I'll get on fixing that.